ShinyExpenseTrackerTests/src/ca/ualberta/cs/shinyexpensetracker/test/TestExpenseItemDetailView.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Test suite to test activity that lets the user view 
+ * information of the selected ExpenseItem
+ * 
+ * Covers Issue 16
+ * @author Oleg Oleynikov 
+ * @version 1.0
+ * @since 2015-03-15
+ */
+
+package ca.ualberta.cs.shinyexpensetracker.test;
+
+import java.math.BigDecimal;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+import ca.ualberta.cs.shinyexpensetracker.ExpenseClaimController;
+import ca.ualberta.cs.shinyexpensetracker.ExpenseItemDetailView;
+import ca.ualberta.cs.shinyexpensetracker.R;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaim;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaimList;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseItem;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseItem.Category;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseItem.Currency;
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.test.ActivityInstrumentationTestCase2;
+import android.widget.TextView;
+import junit.framework.TestCase;
+
+public class TestExpenseItemDetailView extends ActivityInstrumentationTestCase2<ExpenseItemDetailView> {
+
+	public TestExpenseItemDetailView() {
+		super(ExpenseItemDetailView.class);
+	}
+
+	ExpenseItemDetailView activity;
+	ExpenseClaimController controller;
+	
+	
+	protected void setUp() throws Exception {
+		super.setUp();
+		
+		ExpenseClaim claim = new ExpenseClaim("test claim");
+		Calendar newDate = Calendar.getInstance();
+        newDate.set(2000, 00, 01);
+
+		ExpenseItem item = new ExpenseItem("test item", newDate.getTime(), Category.fromString("air fare"), new BigDecimal("0.125"), Currency.CAD, "Test Item", null);
+		item.setHasPhoto(false);
+		claim.addItem(item);
+		ExpenseClaimList claimList = new ExpenseClaimList();
+		claimList.addClaim(claim);
+		// Do things a little backwards to ensure getInstance is singleton
```

Comment:
That was just blind copying from ExpenseClaimControllerTest, since that one was the only implementation of the controller that I got to work in my test